libs/ui/src/ui_strings/app_strings.tsx

```diff
@@ -341,6 +360,14 @@ export const appStrings = {
     </UiString>
   ),
 
+  instructionsBmdContestNavigationPatDevice: () => (
+    <UiString uiStringKey="instructionsBmdContestNavigationPatDevice">
+      To navigate through the contest choices, use the move input. To move to
```

Comment:
"To move" could be confusing with our terminology for the "move" input. What do you think about "To advance to the next contest ..."?

Reply:
Good call - thanks